.github/workflows/release_nightly.yml

```diff
@@ -304,6 +315,17 @@ jobs:
           asset_name: ${{ needs.create-nightly-release.outputs.package_prefix }}-web-extension-firefox.xpi
           asset_content_type: application/x-xpinstall
 
+      - name: Upload Firefox extension (unsigned)
+        if: ${{ steps.sign-firefox.outcome == 'failure' }}
+        uses: actions/upload-release-asset@v1
```

Comment:
Not related to this PR, but it seems that [`actions/upload-release-asset`](https://github.com/actions/upload-release-asset) is unmaintained. Maybe we should replace it with [`action-gh-release`](https://github.com/softprops/action-gh-release)?

Reply:
Same is true for [`actions/create-release`](https://github.com/actions/create-release).

While they work I'm not particularly excited to replace them with something that requires different syntax, though.